dev/how-to/deploy/orchestrated/tidb-in-kubernetes/access-tidb.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Access TiDB Clusters on Kubernetes
+category: how-to
+---
+
+# Access TiDB Clusters on Kubernetes
+
+To access TiDB within a Kubernetes cluster, use the TiDB service domain name `<tidbcluster-name>-tidb.<namespace>`.
+
+To access TiDB outside a Kubernetes cluster, you need to expose TiDB service port. To do that, make the following configuration via the `tidb.service` field in the `values.yaml` file in the tidb-cluster Helm chart.
+
+{{< copyable "" >}}
+
+```yaml
+tidb:
+  service:
+    type: NodePort
+    # externalTrafficPolicy: Cluster
+    # annotations:
+    # cloud.google.com/load-balancer-type: Internal
+```
+
+## NodePort
+
+Without LoadBalancer, expose the TiDB service port in the following two modes of NodePort:
+
+- `externalTrafficPolicy=Cluster`: All machines in the Kubernetes cluster assign a NodePort to TiDB Pod, which is the default mode.
+- `externalTrafficPolicy=Local`: Only those machines that runs TiDB assign NodePort to TiDB Pod so that you can access local TiDB instances.
+
+    When the `Local` mode is in use, it is recommended to enable the `StableScheduling` feature of tidb-scheduler. Tidb-scheduler tries to schedule the newly added TiDB instances to the existing machines during the upgrade process. With such scheduling, client outside Kubernetes cluster does not need to upgrade configuration after TiDB is restarted.
+
+### See the IP/PORT exposed in NodePort mode
+
+To see the Node Port assigned by Service, use the following commands to obtain the Service object of TiDB:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+namespace=<your-tidb-namesapce>
+```
+
+{{< copyable "shell-regular" >}}
+
+```shell
+release=<your-tidb-release-name>
+```
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl -n ${namespace} get svc ${release}-tidb -ojsonpath="{.spec.ports[?(@.name=='mysql-client')].nodePort}{'\n'}"
+```
+
+You might encounter the following two situations when seeing which nodes' IP can access TiDB service:
+
+- When `externalTrafficPolicy` is configured as `Cluster`, IPs of all nodes can access TiDB.
```

Comment:
Please try to understand the meaning first. If you cannot understand it, consult the author of the Chinese document.
```suggestion
- When `externalTrafficPolicy` is configured as `Cluster`, you can use the IP of any node to access TiDB services.
```